### DISCUSSION TITLE
Cannot login after logout

### DISCUSSION THREAD

**Discussion by harold-whistler** [requarks/wiki#3253] [Error / Bug Report]
**Question**
After i logout im going to login page and it display background image and no login form.
Host Info (please complete the following information):

    OS: Ubuntu 20, Docker
    Wiki.js version: 2.5.170
    Database engine: included in docker compose

Additional context
Elastics search attached as search engine.

**Comment by harold-whistler** [comment]
More information 
![Screenshot from 2020-11-20 03-55-49](https://user-images.githubusercontent.com/62359993/99752389-88257b80-2ae4-11eb-90d4-312d8d510699.png)

**Comment by seanecoffey** [comment]
I'm assuming that you're using local authentication.
Did you turn on "Hide Local Authentication Provider" (Under Security>Login)?

If so, this says:

> Don't show the local authentication provider on the login screen. Add ?all to the URL to temporarily use it.

If this is the case, add ?all to the end of your login URL.
